modules/core/04-channel/keeper/packet.go

```diff
@@ -105,34 +106,19 @@ func (k Keeper) SendPacket(
 		}
 	}
 
-	nextSequenceSend, found := k.GetNextSequenceSend(ctx, packet.GetSourcePort(), packet.GetSourceChannel())
-	if !found {
-		return sdkerrors.Wrapf(
-			types.ErrSequenceSendNotFound,
-			"source port: %s, source channel: %s", packet.GetSourcePort(), packet.GetSourceChannel(),
-		)
-	}
-
-	if packet.GetSequence() != nextSequenceSend {
-		return sdkerrors.Wrapf(
-			types.ErrInvalidPacket,
-			"packet sequence ≠ next send sequence (%d ≠ %d)", packet.GetSequence(), nextSequenceSend,
-		)
-	}
-
 	commitment := types.CommitPacket(k.cdc, packet)
 
-	nextSequenceSend++
-	k.SetNextSequenceSend(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), nextSequenceSend)
-	k.SetPacketCommitment(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), packet.GetSequence(), commitment)
+	sequence++
+	k.SetNextSequenceSend(ctx, srcPort, srcChannel, sequence)
+	k.SetPacketCommitment(ctx, srcPort, srcChannel, packet.GetSequence(), commitment)
```

Comment:
Hmm, I'd be careful here if we are also trying to return the sequence in the API, you don't want to make it so the nextSequenceSend is returned

Maybe it makes more sense to do:
```go
k.SetNextSequenceSend(ctx, srcPort, srcChannel, sequence+1)
```

Reply:
Technically you could be fine by ensuring you return packet.GetSequence(), but I think I'd actually prefer not to increment the sequence variable. 

You are using the variable for two different purposes:
- to get the sequence of the packet to send
- to set the next packet sequence to send

I'd rather not just the behaviour unexplicitly partway through the code

Reply:
Can you double check that we verify these values are correctly set in the tests?

Reply:
Done. I ended up using packet.GetSequence() since that is what gets used in the rest of the lines, but I did switch to just setting sequence+1. So future changes don't affect anything.

I also added tests, so it would be caught anyway